articles/overview/overview.md

```diff
@@ -0,0 +1,84 @@
+
+Harnessing the unique behavior of quantum physics and applying it to computing, quantum computers introduce new concepts to traditional programming methods, making use of quantum physics behaviors such as superposition, entanglement, and quantum interference.
+
+## What can a quantum computer do?
+
+A quantum computer isn't a supercomputer that can do everything faster, but there are a few areas where quantum computers do exceptionally well.
+
+- [Quantum simulation](xref:microsoft.quantum.overview.introduction#quantum-simulation)
+- [Cryptography](xref:microsoft.quantum.overview.introduction#cryptography-and-shors-algorithm)
+- [Search](xref:microsoft.quantum.overview.introduction#search-and-grovers-algorithm)
+- [Optimization](xref:microsoft.quantum.overview.introduction#quantum-inspired-computing-and-optimization)
+- [Machine learning](xref:microsoft.quantum.overview.introduction#quantum-machine-learning)
+
+## Quantum simulation
+
+Since quantum computers use quantum phenomena in computation, they are well-suited for modeling other quantum systems. Photosynthesis, superconductivity, and complex molecular formations are examples of quantum mechanisms that quantum programs can simulate.
+
+## Cryptography and Shor’s algorithm
+
+In 1994, Peter Shor showed that a scalable quantum computer could break widely used encryption techniques such as the RSA algorithm. Classical cryptography relies on the intractability of problems such as integer factorization or discrete logarithms, many of which can be solved more efficiently using quantum computers.
+
+## Search and Grover’s algorithm
+
+In 1996, Lov Grover developed a quantum algorithm that dramatically sped up the solution to unstructured data searches, running the search in fewer steps than any classical algorithm could.
+
+> [!NOTE]
+> Ready to try it out? Jump to the Q# tutorial [Implement Grover's search algorithm in Q#](xref:microsoft.quantum.quickstarts.search).
+
+## Quantum-inspired computing and optimization
+
+Quantum-inspired algorithms use quantum principles for increased speed and accuracy but implement on classical computer systems. This approach allows developers to leverage the power of new quantum techniques today without waiting for quantum hardware, which is still an emerging industry.
+
+Optimization is the task of balancing a system for maximum efficiency and minimum cost and is a crucial issue for industry and science. Quantum-inspired optimization algorithms running on today's classical computers can find solutions that up to now have not been possible. In addition to optimizing traffic flow to reduce congestion, there is airplane gate assignment, package delivery, job scheduling and more. With breakthroughs in materials science, there will be new forms of energy, batteries with larger capacity, and lighter and more durable materials.
```

Comment:
Not all optimization tasks seek to increase efficiency of a process; while that is an important example, it's one example of what optimization problems can look like.